setup.py

```diff
@@ -19,7 +19,7 @@
     version=__version__,
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    python_requires=">=3.12",
+    python_requires=">=3.13",
```

Comment:
From my reading, homeassistant still supports 3.12 and still tests against it 

See

https://github.com/home-assistant/core/blob/24f7bae5f260c7b4417e0b21d4f742ca916b2349/pyproject.toml#L25

https://github.com/home-assistant/core/blob/24f7bae5f260c7b4417e0b21d4f742ca916b2349/.github/workflows/ci.yaml#L45